runtime/refcountable_php_classes.h

```diff
@@ -117,11 +139,16 @@ class refcountable_php_classes  : public ManagedThroughDlAllocator {
   }
 
   void set_refcnt(uint32_t new_refcnt) noexcept {
-    refcnt = new_refcnt;
+    refcnt_ = new_refcnt;
+  }
+
+  uint32_t &get_unique_index_ref() noexcept {
+    return unique_index_;
   }
 
 private:
-  uint32_t refcnt{0};
+  uint32_t refcnt_{0};
+  uint32_t unique_index_{0};
```

Comment:
Why does it not change size of every php class? As far as I can see, alignment of `refcountable_php_classes` is 4

Reply:
You are right, but most of our data types:
int64_t - 8 bytes
double - 8 bytes
class_instance - 8 bytes
mixed - align 8 bytes , size 16
string - 8 bytes
array - 8 bytes

and only bool - 1 byte, and Optional<bool> - 2 bytes

so it matters only for the instances with bool or Optional<bool> (or tuple<bool, bool> :)) ) as the first fields, and for the other cases it doesn't matter at all.

Reply:
ok, I got it)